pkg/gce-pd-csi-driver/utils.go

```diff
@@ -28,7 +28,8 @@ import (
 )
 
 const (
-	fsTypeXFS = "xfs"
+	fsTypeXFS  = "xfs"
+	maxLogChar = 10000
```

Comment:
Do you think it's not too messy to plumb this limit through with a command-line flag? Then it it turns out 10k is still too large we could easily ratchet it down with just a yaml tweak and not a code change + version release.

Reply:
done